oak-run/src/test/java/org/apache/jackrabbit/oak/index/DocumentStoreIndexerIT.java

```diff
@@ -214,6 +214,16 @@ protected CompositeIndexer prepareIndexers(NodeStore nodeStore, NodeBuilder buil
 
     }
 
+    @Test
+    public void testParallelIndexing() throws Exception {
+        try {
+            System.setProperty(FlatFileNodeStoreBuilder.OAK_INDEXER_PARALLEL_INDEX, "true");
+            bundling();
+        } finally {
+            System.clearProperty(FlatFileNodeStoreBuilder.OAK_INDEXER_PARALLEL_INDEX);
+        }
+    }
+
```

Comment:
you could use the utility introduced in https://github.com/apache/jackrabbit-oak/pull/595 to set/clean system properties